demo/config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "EthereumNodeURL": "ws://127.0.0.1:8545",
+	"Mnemonic": "pistol kiwi shrug future ozone ostrich match remove crucial oblige cream critic",
+	"EnclaveDerivationPath": "m/44'/60'/0'/0/0",
```

Comment:
why not use /3 or higher for the enclave instead of a shared key with the op?

Reply:
ahh I think we hardcode the enclave wallet derivation path to /0 right now, right?